styleguide/components/pagination/_basic.html

```diff
@@ -0,0 +1,14 @@
+<ul class="cads-paging">
```

Comment:
I think we should wrap pagination in `nav` elements with `role="navigation"`. Thinking about that also prompted me to look up best practices around pagination accessibility and there's also recommendations to use `aria-current="page"` and a label such as `aria-label="Page three, current page"` on the active element and `aria-label="Pagination Navigation"` on the nav element.

What do you think about improving the documentation to include accessibility features?